lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2519,6 +2519,9 @@ def test_get_students_features(self):
         Test that some minimum of information is formatted
         correctly in the response to get_students_features.
         """
+        for student in self.students:
```

Comment:
I agree with the decision that it's not worth tackling that sort of integration test.